docs/docs/target/container_image.md

```diff
@@ -373,48 +161,45 @@ Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
 </details>
 
 ### OCI Layout
-Trivy supports image directories compliant with [Open Container Image Layout Specification](https://github.com/opencontainers/image-spec/blob/master/spec.md).
+Trivy supports scanning image filesystem directories compliant with [Open Container Image Layout Specification](https://github.com/opencontainers/image-spec/blob/master/spec.md).
+
+Examples:
 
 Buildah:
 
-```
+```bash
 $ buildah push docker.io/library/alpine:3.11 oci:/path/to/alpine
 $ trivy image --input /path/to/alpine
 ```
 
 Skopeo:
 
-```
+```bash
 $ skopeo copy docker-daemon:alpine:3.11 oci:/path/to/alpine
 $ trivy image --input /path/to/alpine
 ```
 
 Referencing specific images can be done by their tag or by their manifest digest:
-```
+
+```bash
 # Referenced by tag
 $ trivy image --input /path/to/alpine:3.15
 
 # Referenced by digest
 $ trivy image --input /path/to/alpine@sha256:82389ea44e50c696aba18393b168a833929506f5b29b9d75eb817acceb6d54ba
 ```
 
-## SBOM
-Trivy supports the generation of Software Bill of Materials (SBOM) for container images and the search for SBOMs during vulnerability scanning.
+## SBOM Discovery
+Trivy can discover SBOM for the scanned image, and use it for vulnerability scanning instead of analyzing the image itself. This can speed up and simplify repeated scans of the same image.
 
-### Generation
```

Comment:
We got a request from the community to add a section about SBOM generation in this page so that people will not misunderstand Trivy supports only SBOM discovery.

Reply:
I could add it again but it would need to appear on every target, and if we plan to do https://github.com/aquasecurity/trivy/issues/5183 (I think we should) then I don't think it's necessary.

Reply:
I converted the issue to discussion as we saw some challenges.
https://github.com/aquasecurity/trivy/discussions/8599#discussioncomment-12599786

Reply:
This is still relevant. I gave it some thought, and although in Trivy we treat SBOM generation simply as one type of output format, some users might not actually perceive it that way. Since it’s just a matter of adding a few lines to include a link to the SBOM page, it isn’t overly verbose, so I don’t think there are any drawbacks.

Reply:
5f05f4aa700cd967727cf6c3f8a5df5ab5a859d1